robolectric/src/main/java/org/robolectric/android/internal/ParallelUniverse.java

```diff
@@ -319,4 +334,30 @@ private static void setUpPackageStorage(ApplicationInfo applicationInfo) {
           tempDirectory.createIfNotExists("deviceDataDir").toAbsolutePath().toString();
     }
   }
+
+  // TODO move/replace this with packageManager
+  private static void registerBroadcastReceivers(Application application,
```

Comment:
Nice - this really feels like it belongs here, rather than in ShadowApplication, mirrors the Android set up code in ActivityThread.